temporal/api/cloud/identity/v1/message.proto

```diff
@@ -66,6 +66,37 @@ message User {
     // The date and time when the user was created
     google.protobuf.Timestamp created_time = 7;
     // The date and time when the user was last modified
-    // Will not be set if the user has never been modified.
+    // Will not be set if the user has never been modified
     google.protobuf.Timestamp last_modified_time = 8;
 }
+
+message UserGroupSpec {
+    // The name of the group as defined in the customer's IdP (e.g. Google group name in Google Workspace)
+    // The name is immutable. Once set, it cannot be changed
+    string name = 1;
+    // The type of the group.
+    // This field is immutable. Once set, it cannot be changed
+    // Allowed values ["google_workspace"].
+    string type = 2;
```

Comment:
I wonder if just the type is enough... should we add a connection_id too ?
But not sure how the user can know the connection id

Reply:
never mind we can, add support for connection id later.